hrm/department_app/models/department.py

```diff
@@ -0,0 +1,14 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField('Department',max_length=50)
```

Comment:
The department field should not be null or empty.

Reply:
Fixed.

Reply:
This is bad practice to use varchar fields as primary keys. Please remove primary_key=True and configure this field so that it is not possible to store NULLs and empty strings in it. And configure primary keys correctly.